### PR TITLE
Add new translations

### PR DESCRIPTION
**Pull request by Jonas-C**
https://trello.com/c/smt53hOR/121-un%C3%B8dvendig-forbehold-i-bruksvilk%C3%A5r
https://trello.com/c/hl0Ufd8r/110-legge-til-varsel-om-at-dersom-man-sletter-en-mappe-s%C3%A5-sletter-man-ogs%C3%A5-undermapper

### CODE REVIEW

**Review by haattis — APPROVED** [review]
LGTM

File: packages/ndla-ui/src/locale/messages-nn.ts [hunk 1]
```diff
@@ -1013,7 +1013,8 @@ const messages = {
     },
     tags: '{{count}} emneknagg',
     tags_plural: '{{count}} emneknaggar',
-    confirmDeleteFolder: 'Er du sikker på at du vil slette mappa? Denne handlinga kan ikkje endrast.',
+    confirmDeleteFolder:
+      'Er du sikker på at du vil slette mappa? Dersom mappa har undermappar vil disse også slettes. Denne handlinga kan ikkje endrast.',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-nn.ts:1017]:
```suggestion
      'Er du sikker på at du vil slette mappa? Dersom mappa har undermapper vil desse også slettast. Denne handlinga kan ikkje endrast.',
```

File: packages/ndla-ui/src/locale/messages-nn.ts [hunk 2]
```diff
@@ -1035,6 +1036,9 @@ const messages = {
       myPage: 'Mi side',
       deleteAccount: 'Slett Min NDLA',
       logout: 'Logg ut av Min NDLA',
+      loginText:
+        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriver noko støtende eller personsensitiv informasjon eller anna persondata i tekstfelt. Les vår ',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-nn.ts:1040]:
```suggestion
        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriv noko støtande eller personsensitiv informasjon eller anna persondata i tekstfelt. Les vår ',
```

**Review by gunnarvelle — COMMENTED** [review]
Språkvask

File: packages/ndla-ui/src/locale/messages-nb.ts [hunk 3]
```diff
@@ -1034,6 +1035,9 @@ const messages = {
       myPage: 'Min side',
       deleteAccount: 'Slett Min NDLA',
       logout: 'Logg ut av Min NDLA',
+      loginText:
+        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikke skriver noe støtende eller personsensitiv informasjon eller annen persondata i tekstfelt. Les vår ',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-nb.ts:1020]:
```suggestion
        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikke skriver noe støtende, personsensitiv informasjon eller andre persondata i tekstfelt. Les vår ',
```

File: packages/ndla-ui/src/locale/messages-nn.ts [hunk 4]
```diff
@@ -1035,6 +1036,9 @@ const messages = {
       myPage: 'Mi side',
       deleteAccount: 'Slett Min NDLA',
       logout: 'Logg ut av Min NDLA',
+      loginText:
+        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriv noko støtande eller personsensitiv informasjon eller anna persondata i tekstfelt. Les vår ',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-nn.ts:1021]:
```suggestion
        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriv noko støtande, personsensitiv informasjon eller andre persondata i tekstfelt. Les vår ',
```

File: packages/ndla-ui/src/locale/messages-se.ts [hunk 5]
```diff
@@ -1035,6 +1036,9 @@ const messages = {
       myPage: 'Min side',
       deleteAccount: 'Slett Min NDLA',
       logout: 'Logg ut av Min NDLA',
+      loginText:
+        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriver noko støtende eller personsensitiv informasjon eller anna persondata i tekstfelt. Les vår ',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-se.ts:1020]:
```suggestion
        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriver noko støtende, personsensitiv informasjon eller andre persondata i tekstfelt. Les vår ',
```

File: packages/ndla-ui/src/locale/messages-sma.ts [hunk 6]
```diff
@@ -1035,6 +1036,9 @@ const messages = {
       myPage: 'Min side',
       deleteAccount: 'Slett Min NDLA',
       logout: 'Logg ut av Min NDLA',
+      loginText:
+        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriver noko støtende eller personsensitiv informasjon eller anna persondata i tekstfelt. Les vår ',
```
Comment (gunnarvelle) [inline packages/ndla-ui/src/locale/messages-sma.ts:1021]:
```suggestion
        'Logg på med Feide for å få tilgang til Min NDLA. Vi ber om at du ikkje skriver noko støtende, personsensitiv informasjon eller andre persondata i tekstfelt. Les vår ',
```

**Review by gunnarvelle — COMMENTED** [review]
Synes dette er litt bedre flyt i setningene.

**Pull request merged**